graylog2-server/src/test/java/org/graylog2/shared/rest/resources/ProxiedResourceTest.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * This file is part of Graylog.
+ *
+ * Graylog is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Graylog is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.graylog2.shared.rest.resources;
+
+import org.graylog2.shared.rest.resources.ProxiedResource.MasterResponse;
+import org.junit.Test;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class ProxiedResourceTest {
+    @Test
+    public void masterResponse() {
```

Comment:
What are we testing here? I does not make sense to test AutoValue. If we are testing `create`, this implies that we are unsure about the parameter type/order of the factory method, which means that we should just use a builder instead.

Reply:
We are testing that the `#body()` method returns the correct value depending on the value of `#entity()`.